python/utils.py

```diff
@@ -214,9 +213,11 @@ def _transform_args_have_changed(
         # Lastly check if any of the transform arg attrs have change
         # if so, transform needs versioning
         db_arg = db_args_w_name[0]
-        if db_arg.description != yaml_arg['description'] or \
-                db_arg.is_optional != yaml_arg.get('is_optional', False) or \
-                db_arg.type != yaml_arg['type']:
+        if (
+            db_arg.description != yaml_arg['description']
+            or db_arg.is_optional != yaml_arg.get('is_optional', False)
+            or db_arg.type != yaml_arg['type']
+        ):
```

Comment:
Wow never knew you could organize long `if` statements like this.

Like that way more than using the slashes